src/main/java/io/fusionauth/http/server/Configurable.java

```diff
@@ -77,6 +77,8 @@ default T withContextPath(String contextPath) {
    * @param validator The validator.
    * @return This.
    */
+  // TODO : Daniel : Review : It would be cool to offer some additional methods such as withOptionalExpectValidator?
+  //                 Or would there be another way that we could allow the caller to designate null as don't set?
```

Comment:
I initially was not going to allow this to be set to `null`. I am still thinking that is maybe the better option. 

This would means we force you to be explicit and provide an expect handler. If it is `null` then there is a "default" behavior in the code instead of it being prescribed. 

The reason I took it out initially was we had a test that just passed in `null` and expected it to set this to the default behavior - hence the comment about the `withOptional`. 

Leaning towards requiring a non-null value here.